en/docs/learn/integration-tutorials/transforming-message-content.md

```diff
@@ -50,129 +50,60 @@ The client message format must be transformed to the back-end service message fo
 
```

Comment:
In this tutorial, you send a request message to a back-end service where the format of the request payload is different to what is expected by the back-end service.  ->

In this tutorial, you will send a request message to a back-end service where the format of the request payload differs from what the back-end service expects.